contributors/devel/cherry-picks.md

```diff
@@ -19,26 +19,45 @@ task is for backporting PRs from master to release branches.
    github.com/github/hub` assuming you have a standard golang development
    environment.
 
-## Initiate a Cherry Pick
- * Run the [cherry pick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
+## Initiate a Cherrypick
+ * Run the [cherrypick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
    This example applies a master branch PR #98765 to the remote branch
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
+   * Be aware the cherrypick script assumes you have a git remote called `upstream` that points at the Kubernetes github org.
+   Please see our [recommended Git workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
+   * You will need to run the cherrypick script separately for each patch release you want to cherrypick to.
+
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestones on cherrypick PRs should be the milestone for the target release branch (e.g.milestone 1.11 for a cherrypick onto release-1.11).
```

Comment:
s/`e.g.milestone`/`e.g., milestone`

Reply:
regarding your first question, the patch release manager applies the `cherrypick-approved` label.

Reply:
oh I see your question now - sorry! 😓 I'm honestly not sure, will ask.

Reply:
> AFAIK the patch release manager applies the cherrypick-approved label.

Yes, this is correct. :)

The patch release manager is responsible for approving cherrypicks.